inverse/beamformer_lcmv.m

```diff
@@ -168,8 +169,9 @@
   tmplambda = 0;
 end
 
-if projectnoise
+if projectnoise || strcmp(weightnorm, 'nai')
```

Comment:
@robertoostenveld Here, I am not sure, whether it would be necessary to inform the user via a warning, that noise is computed.

Reply:
I think it is not necessary. Also, the noise is computed and used for the weight normalisation, but not outputted

Reply:
That makes sense!